<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <group ns="X4">   
    <node pkg="msfm3d" type="goal_pose_sampler" name="goal_pose_sampler" output="screen">
      <!-- Params -->
      <param name="update_rate" value="2.0" />
    </node>
  </group>
</launch>